osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -127,20 +127,7 @@ public int RepeatCount
         /// If <see langword="false"/>, <see cref="Slider"/>'s judgement is fully handled by its nested <see cref="HitObject"/>s.
         /// If <see langword="true"/>, this <see cref="Slider"/> will be judged proportionally to the number of nested <see cref="HitObject"/>s hit.
         /// </summary>
-        public bool ClassicSliderBehaviour
-        {
-            get => classicSliderBehaviour;
-            set
-            {
-                classicSliderBehaviour = value;
-                if (HeadCircle != null)
-                    HeadCircle.ClassicSliderBehaviour = value;
-                if (TailCircle != null)
-                    TailCircle.ClassicSliderBehaviour = value;
-            }
-        }
-
-        private bool classicSliderBehaviour;
+        public bool ClassicSliderBehaviour { get; set; }
```

Comment:
The xmldoc of this flag needs an update. And probably so does the naming in only place that sets it, which is the classic mod "no slider head accuracy" toggle. I'm not sure how to even describe what this flag does now, but to call it "no slider head accuracy" seems... inaccurate? The description of that setting is 100% wrong now, since sliders are now proportionally judged regardless of the flag, just with a new cap factor from the slider head in non-classic.

Reply:
I've adjusted the xmldoc, kinda hard to describe..